ui/pages/login/index.js

```diff
@@ -2,6 +2,7 @@ import { useRouter } from 'next/router'
 import { useState } from 'react'
 import useSWR, { useSWRConfig } from 'swr'
 import { useServerConfig } from '../../lib/serverconfig'
+import Cookies from 'universal-cookie'
```

Comment:
can we rearrange the import orders so it is consistent through the app? 

`[external libraries import]

[internal libraries import]

[components]
`